{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "/Users/alexrutherford/repos/pymarl/src",
    "dependencies": [
      "munch==2.5.0",
      "numpy==1.21.5",
      "PyYAML==6.0",
      "sacred==0.8.2",
      "torch==1.10.0"
    ],
    "mainfile": "main.py",
    "name": "pymarl",
    "repositories": [
      {
        "commit": "7e26ba46ce2d868b2e57d7375491a783da0e626f",
        "dirty": true,
        "url": "git@github.com:am-rutherford/pymarl.git"
      },
      {
        "commit": "7e26ba46ce2d868b2e57d7375491a783da0e626f",
        "dirty": true,
        "url": "git@github.com:am-rutherford/pymarl.git"
      },
      {
        "commit": "7e26ba46ce2d868b2e57d7375491a783da0e626f",
        "dirty": true,
        "url": "git@github.com:am-rutherford/pymarl.git"
      }
    ],
    "sources": [
      [
        "main.py",
        "_sources/main_ac85fb47de40ce98de2a16a2fd6ccae9.py"
      ],
      [
        "run.py",
        "_sources/run_95ad78a4fe977c885b6a7fc6e626bb88.py"
      ],
      [
        "utils/logging.py",
        "_sources/logging_aa14e56e60b01dd7a9ddb8e6cdf36b85.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/Users/alexrutherford/opt/anaconda3/envs/rapport/lib/python3.7/site-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"src/main.py\", line 35, in my_main\n    run(_run, config, _log)\n",
    "  File \"/Users/alexrutherford/repos/pymarl/src/run.py\", line 47, in run\n    run_sequential(args=args, logger=logger)\n",
    "  File \"/Users/alexrutherford/repos/pymarl/src/run.py\", line 145, in run_sequential\n    learner.load_models(model_path)\n",
    "  File \"/Users/alexrutherford/repos/pymarl/src/learners/q_learner.py\", line 150, in load_models\n    self.mac.load_models(path)\n",
    "  File \"/Users/alexrutherford/repos/pymarl/src/controllers/basic_controller.py\", line 71, in load_models\n    self.agent.load_state_dict(th.load(\"{}/agent.th\".format(path), map_location=lambda storage, loc: storage))\n",
    "  File \"/Users/alexrutherford/opt/anaconda3/envs/rapport/lib/python3.7/site-packages/torch/nn/modules/module.py\", line 1483, in load_state_dict\n    self.__class__.__name__, \"\\n\\t\".join(error_msgs)))\n",
    "RuntimeError: Error(s) in loading state_dict for RNNAgent:\n\tsize mismatch for fc1.weight: copying a param with shape torch.Size([64, 11]) from checkpoint, the shape in current model is torch.Size([64, 13]).\n"
  ],
  "heartbeat": "2022-02-24T23:02:52.659361",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Core(TM) i7-8559U CPU @ 2.70GHz",
    "hostname": "Alexs-MacBook-Pro-2.local",
    "os": [
      "Darwin",
      "Darwin-20.6.0-x86_64-i386-64bit"
    ],
    "python_version": "3.7.10"
  },
  "meta": {
    "command": "my_main",
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2022-02-24T23:02:52.582080",
  "status": "FAILED",
  "stop_time": "2022-02-24T23:02:52.660468"
}